iib/workers/tasks/build.py

```diff
@@ -785,6 +941,28 @@ def handle_add_request(
     _update_index_image_build_state(request_id, prebuild_info)
 
     with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
+        if from_index:
+            msg = 'Checking if bundles are already present in index image'
+            log.info(msg)
+            set_request_state(request_id, 'in_progress', msg)
+            present_bundles = _get_present_bundles(from_index, temp_dir)
+            filtered_bundles = _get_missing_bundles(present_bundles, resolved_bundles)
+            excluded_bundles = [b for b in resolved_bundles if b not in filtered_bundles]
```

Comment:
Can we please use `bundle` instead of `b`. I'm usually not a fan of single letter variable names unless absolutely required.

Reply:
done